### DISCUSSION TITLE
Github Actions trying to use Github Packages

### DISCUSSION THREAD

**Discussion by agamorzywolek** [community/community#54540] [Actions]
### Select Topic Area

Question

### Body

Hello,

My question is related to Github Actions and Github Packages.
 
I have created a Github Actions workflow with permissions "packages: write". Workflow is responsible for building a docker image of the dot net application. 

On step "dotnet restore" - I am getting:
```
#41 5.039   Retrying 'FindPackagesByIdAsync' for source 'https://nuget.pkg.github.com/ORG/download/XXXXXXX/index.json'.
#41 5.039   Response status code does not indicate success: 403 (Forbidden).
#41 5.059 /usr/share/dotnet/sdk/7.0.203/NuGet.targets(132,5): warning : Your request could not be authenticated by the GitHub Packages service. Please ensure your access token is valid and has the appropriate scopes configured. 
```
Sometimes the whole job fails because of that, sometimes it passes. The behavior is purely random. 

To build and push docker image step I pass: 
```
        secrets: |
          "github_token=${{ secrets.GITHUB_TOKEN }}"
```

I picked the repository (where the workflow is) to access this package using Github Actions in the package's settings and I gave the role admin. 

Is there anything more that must be done to get this working?

**Comment by jsoref** [comment]
File a ticket to https://support.github.com/
